server/channels/api4/work_templates.go

```diff
@@ -21,24 +21,24 @@ func areWorkTemplatesEnabled(c *Context) *model.AppError {
 	if !c.App.Config().FeatureFlags.WorkTemplate {
 		return model.NewAppError("areWorkTemplatesEnabled", "api.work_templates.disabled", nil, "feature flag is off", http.StatusNotFound)
 	}
-
-	// we have to make sure that playbooks plugin is enabled and board is a product
-	pbActive, err := c.App.IsPluginActive(model.PluginIdPlaybooks)
-	if err != nil {
-		return model.NewAppError("areWorkTemplatesEnabled", "api.work_templates.disabled", nil, "", http.StatusInternalServerError).Wrap(err)
-	}
-	if !pbActive {
-		return model.NewAppError("areWorkTemplatesEnabled", "api.work_templates.disabled", nil, "playbook plugin not active", http.StatusNotFound)
-	}
```

Comment:
This is temporary here given that mono-repo doesn't have Playbook as a plugin but v7.10 does

Reply:
Yeah it should be good.